test/project_types/script/commands/enable_test.rb

```diff
@@ -220,6 +217,40 @@ def test_calls_application_enable_with_configuration_file_and_properties_overrid
         end
       end
 
+      def test_calls_application_enable_with_invalid_configuration_file
+        File.open("enable_config_file.yml", "w+") { |file| file.write("key1 \"value1\"\nkey2: \"value2\"") }
+
+        Script::UI::ErrorHandler
+          .expects(:display)
+          .with(
+            failed_op: nil,
+            cause_of_error: ShopifyCli::Context.message('script.error.invalid_config', "enable_config_file.yml"),
+            help_suggestion: nil,
+          )
+
+        assert_raises ShopifyCli::AbortSilent do
```

Comment:
I tested this locally and this is actually giving me flaky tests as a `PStore::Error` is raised instead of `AbortSilent`. Could we look into it? We definitely don't need more of those 🙈

Reply:
@ErinRenNumber1 and I both ran the tests 10 times each and weren't able to reproduce this error, I think the test you suggested above fixed the problem.

Reply:
Perfect, looks like its good on my end too now. Thanks for checking!